webapp/channels/src/components/channel_banner/channel_banner.tsx

```diff
@@ -29,7 +31,10 @@ type Props = {
 
 export default function ChannelBanner({channelId}: Props) {
     const channelBannerInfo = useSelector((state: GlobalState) => getChannelBanner(state, channelId));
-    const showChannelBanner = useSelector((state: GlobalState) => selectShowChannelBanner(state, channelId));
+    const license = useSelector(getLicense);
+    const licenseEnabled = isMinimumEnterpriseAdvancedLicense(license);
```

Comment:
Can we not update the license check in `selectShowChannelBanner`?

Reply:
Same reason as: https://github.com/mattermost/mattermost/pull/34212#discussion_r2557318366